
int32 from_drone_id
int32 to_drone_id

float64 stamp
int8[] ego_ids
int8[] other_ids

# FAME
geometry_msgs/Point ego_posit
geometry_msgs/Point other_posit
